templates/mail/import_vaccination_notification.txt

```diff
@@ -0,0 +1,17 @@
+{% autoescape off %}
+
+Opération effectuée:
+- {{imported}} centres importés
+- {{skipped}} écartés
+
+{%if len(errors) %}
```

Comment:
Nit: please ensure consistent var and template tags formatting, they should have a leading and trailing spaces, eg. `{% foo %}` and `{{ foo }}`, not `{%foo %}` or `{{foo}}`.